.travis.yml

```diff
@@ -220,24 +241,27 @@ matrix:
                   | capture(\"runtime is (?<runtime>[0-9]+)\").runtime" \
               || echo 0)
 
-          STATUSM="Passed, runtime is ${CURR} cycles"
-          if [ "$PREV" -ne 0 ]
-          then
-              STATUSM="$STATUSM ($(python -c "print '%+d' % ($CURR-$PREV)") cycles)"
-          fi
-        - bash -c "$STATUS" success "$STATUSM"
+          #STATUSM="Passed, runtime is ${CURR} cycles"
+          #if [ "$PREV" -ne 0 ]
+          #then
+          #    STATUSM="$STATUSM ($(python -c "print '%+d' % ($CURR-$PREV)") cycles)"
+          #fi
```

Comment:
Can we drop these comments?

Reply:
Ah! That's what I missed.

I think these are still needed (@geky to comment), but I meant to update the bash variables and these jobs.